definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -1983,6 +1983,24 @@
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/update-task-list/submitted",
     "EndButtonLabel": "Save and continue",
     "EventEnablingCondition": "caseTypeOfApplication=\"C100\" AND taskListVersion=\"v3\""
+  },
+  {
+    "LiveFrom": "26/07/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "removeDraftOrder",
+    "Name": "Remove draft order",
+    "Description": "Remove draft order",
+    "DisplayOrder": 32,
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-remove-draft-order-dropdown",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/remove-draft-order/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/remove-draft-order/submitted",
+    "PreConditionState(s)": "AWAITING_RESUBMISSION_TO_HMCTS;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
+    "PostConditionState": "ALL_FINAL_ORDERS_ISSUED(doesOrderClosesCase=\"Yes\"):1;*",
```

Comment:
Precondition should be : CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME 

Postcondition should be : *

The idea is , This event should not change the state of the case .